sdk/digitaltwins/azure-digitaltwins/azure/digitaltwins/_digitaltwins_client.py

```diff
@@ -0,0 +1,547 @@
+        :return: None
+        :rtype: None
+        :raises: ~azure.core.exceptions.HttpResponseError
+        """
+        if not message_id:
+            message_id = uuid.UUID
+        timestamp = datetime.now
+        return self._client.digital_twins.send_telemetry(
+            digital_twin_id,
+            message_id,
+            telemetry=payload,
+            dt_timestamp=timestamp,
+            **kwargs
+        )
+
+    def publish_component_telemetry(self, digital_twin_id, component_path, payload, message_id=None, **kwargs):
+        # type: (str, str, object, str, **Any) -> None
+        """Publish telemetry from a digital twin's component, which is then consumed by
+            one or many destination endpoints (subscribers) defined under.
+        Status codes:
+        200 (OK): Success.
+        400 (Bad Request): The request is invalid.
+        404 (Not Found): There is either no digital twin with the provided id or the component path is
+        invalid.
+
+        :param digital_twin_id: The Id of the digital twin.
+        :type digital_twin_id: str
+        :param component_path: The name of the DTDL component.
+        :type component_path: str
+        :param payload: The telemetry payload to be sent.
+        :type payload: object
+        :param message_id: The message Id.
+        :type message_id: str
+        :keyword str content_type: Content type to set in the HTTP header. Default to "application/json".
+        :return: None
+        :rtype: None
+        :raises: ~azure.core.exceptions.HttpResponseError
+        """
+        if not message_id:
+            message_id = uuid.UUID
+        timestamp = datetime.now
+           
+        return self._client.digital_twins.send_component_telemetry(
+            digital_twin_id,
+            component_path,
+            dt_id=message_id,
+            telemetry=payload,
+            dt_timestamp=timestamp,
+            **kwargs
+        )
+
+    def get_model(self, model_id, include_model_definition=False, **kwargs):
+        # type: (str, bool, **Any) -> "models.ModelData"
+        """Get a model, including the model metadata and the model definition.
+        Status codes:
+        200 (OK): Success.
+        404 (Not Found): There is no model with the provided id.
+
+        :param model_id: The Id of the model.
+        :type model_id: str
+        :param include_model_definition: When true the model definition
+            will be returned as part of the result.
+        :type include_model_definition: bool
+        :returns: The ModelDate object.
+        :rtype: ~azure.digitaltwins.models.ModelData
+        :raises :class: `~azure.core.exceptions.HttpResponseError`
+        """
+        return self._client.digital_twin_models.get_by_id(model_id, include_model_definition, **kwargs)
+
+    def list_models(self, dependencies_for, **kwargs):
+        # type: (str, bool, int, **Any) -> Iterable["models.PagedModelDataCollection"]
```

Comment:
This one should also hide the collection object - so just `ItemPaged[ModelData]`

Reply:
Fixed